packages/itwinui-css/backstop/tests/color.html

```diff
@@ -56,12 +56,24 @@
       .demo-background-positive-hover { background-color: var(--iui-color-background-positive-hover); }
       .demo-background-warning-hover { background-color: var(--iui-color-background-warning-hover); }
       .demo-background-negative-hover { background-color: var(--iui-color-background-negative-hover); }
-      .demo-hover-backgrounds {
+      .demo-hover-backgrounds,
+      .demo-data-visualization {
         list-style: none;
         padding: 0;
         display: flex;
         flex: 1;
       }
+      .demo-data-visualization-skyblue { background-color: var(--iui-color-background-skyblue); }
+      .demo-data-visualization-celery { background-color: var(--iui-color-background-celery); }
+      .demo-data-visualization-froly { background-color: var(--iui-color-background-froly); }
+      .demo-data-visualization-steelblue { background-color: var(--iui-color-background-steelblue); }
+      .demo-data-visualization-sunglow { background-color: var(--iui-color-background-sunglow); }
+      .demo-data-visualization-seabuckthorn { background-color: var(--iui-color-background-seabuckthorn); }
+      .demo-data-visualization-montecarlo { background-color: var(--iui-color-background-montecarlo); }
+      .demo-data-visualization-poloblue { background-color: var(--iui-color-background-poloblue); }
+      .demo-data-visualization-bouquet { background-color: var(--iui-color-background-bouquet); }
+      .demo-data-visualization-ash { background-color: var(--iui-color-background-ash); }
+      .demo-data-visualization-oak { background-color: var(--iui-color-background-oak); }
```

Comment:
Would the soft-background colors be limited to just `avatar` and `badge`? Or would we use them in future components as well?

Reply:
So far iTwinUI only uses it in avatar & badge.  It could be used in future iTwinUI components but it could also be used by non-iTwinUI components like a custom component build by a dev team.